### PR TITLE
bpo-29591: [2.7] Update VS project files

### PR DESCRIPTION
**Pull request by jkloth**
Updates the Visual Studio project files to fix regressions due to Expat 2.2.0
* Silences new warnings when building with MSBuild
* Fixes #include error when building with vcbuild

### CODE REVIEW

File: PC/VS9.0/pyproject.vsprops [hunk 1]
```diff
@@ -12,7 +12,7 @@
 		InlineFunctionExpansion="1"
 		EnableIntrinsicFunctions="true"
 		AdditionalIncludeDirectories="..\..\Include; ..\..\PC"
-		PreprocessorDefinitions="_WIN32"
+		PreprocessorDefinitions="_WIN32;WIN32"
```
Comment (vstinner) [inline PC/VS9.0/pyproject.vsprops:15]: I'm not sure that this change is still needed with expat 2.2.1. Would you mind to check please?

**Review by vstinner — COMMENTED** [review]
Can you please elaborate the commit message to list removed define and explain that they are now already defined in Modules/expat/winconfig.h?

**Comment by vstinner** [comment]
The overall change LGTM, except of my (minor) comments. Can you please rebase it since I just merged my expat 2.2.1 change?

File: PC/VS9.0/pyproject.vsprops (hunk 1 above)
Comment (jkloth) [reply PC/VS9.0/pyproject.vsprops:15]: Actually this is unrelated to just Expat.  It brings the VS9.0 project files inline with the MSBuild ones.  Ever wonder why the Expat change "just worked" on the other releases? ;)

**Comment by jkloth** [comment]
Rebased and modified commit message.

File: PC/VS9.0/pyproject.vsprops (hunk 1 above)
Comment (vstinner) [reply PC/VS9.0/pyproject.vsprops:15]: Ok fine. I'm just surprised that we have to define it manualy. It should already be defined by <windows.h> no?

**Comment by vstinner** [comment]
The ch1nge looks safe and should at least fix the HAVE_MEMMOVE warning. Thanks.

**Pull request merged**